client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/screen-main.tsx

```diff
@@ -98,27 +76,21 @@ const ScreenMain = ( {
 							checked={ hasSections }
 							icon={ layout }
 							aria-label={ translate( 'Sections' ) }
-							onClick={ () => handleNavigatorItemSelect( 'section', INITIAL_CATEGORY ) }
-							active={ shouldOpenCategoryList }
+							onClick={ () =>
+								handleNavigatorItemSelect( 'section', NAVIGATOR_PATHS.SECTIONS, INITIAL_CATEGORY )
```

Comment:
We can now remove the `hasNestedItems` prop below (and its logic in `NavigatorItem`) :)